airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/DebeziumRecordIterator.java

```diff
@@ -53,17 +53,13 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DebeziumRecordIterator.class);
 
-  /**
-   * This is not private and final because we need to override in tests otherwise each test would
-   * continue to run for 5 minutes
-   */
-  static TimeUnit sleepTimeUnit = TimeUnit.MINUTES;
   private static final int SLEEP_TIME_AMOUNT = 5;
```

Comment:
this would 100% trip me up because I would first go: "what is the time unit for this" and then realize it's actually being used as two different constants. Could we instead make it two different constants e.g: 

```
private static final int FIRST_RECORD_WAIT_TIME_MINUTES = 5;
private static final int SUBESQUENT_RECORD_WAIT_TIME_SECONDS = 5;
```